src/routes/transactions/mappers/common/safe-app-info.mapper.ts

```diff
@@ -29,7 +29,7 @@ export class SafeAppInfoMapper {
 
     const [safeApp] = await this.safeAppsRepository.getSafeApps({
       chainId,
-      clientUrl: originUrl,
+      url: originUrl,
```

Comment:
Let's add test coverage for this.

Reply:
There's already a test for this, but it's bogus as it was returning false positives. I don't have time right now to fix it. Feel free to make a separate PR. This fix needs to go live ASAP.

Reply:
Understood. I'll create a separate PR later on.